### PR TITLE
add more details about resource usage

### PR DESCRIPTION
**Pull request by ThomasWaldmann**
not sure how to write these formulas better, right now they maybe look worse than what's usually happening in practice.

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -206,44 +206,88 @@ Resource Usage
...
+Repository index (server only):
+    Proportional to the amount of data chunks in your repo. Lots of chunks
+    in your repo imply a big repository index.
+    It is possible to tweak the chunker params (see create options) to
+    influence the amount of chunks being created.
+
+Temporary files (client):
+    Reading data and metadata from a FUSE mounted repository will consume about
+    the same space as the deduplicated chunks used to represent them in the
+    repository.
```
Comment (enkore) [inline docs/usage.rst]: No, the cache doesn't have all chunks, only small ones.

What it does need is as much free space as the size of the metadata, which can be a couple gigs for large archives...

File: docs/usage.rst [hunk 2]
```diff
@@ -206,44 +206,88 @@ Resource Usage
...
     If your repository is remote, all deduplicated (and optionally compressed/
     encrypted) data of course has to go over the connection (ssh: repo url).
     If you use a locally mounted network filesystem, additionally some copy
     operations used for transaction support also go over the connection. If
     you backup multiple sources to one target repository, additional traffic
     happens for cache resynchronization.
 
-In case you are interested in more details, please read the internals documentation.
+In case you are interested in more details (like formulas), please read the
+internals documentation.
```
Comment (enkore) [inline docs/usage.rst]: link?

File: docs/usage.rst [hunk 3]
```diff
@@ -206,44 +206,88 @@ Resource Usage
...
+Chunks index (client only):
     Proportional to the amount of data chunks in your repo. Lots of chunks
     in your repo imply a big chunks index.
     It is possible to tweak the chunker params (see create options).
 
-Files index:
-    Proportional to the amount of files in your last backup. Can be switched
+Files index (client only):
+    Proportional to the amount of files in your last backups. Can be switched
     off (see create options), but next backup will be much slower if you do.
```
Comment (ThomasWaldmann) [inline docs/usage.rst]: How about "... might be much slower" and also documenting that "the speed benefit of using the files cache is proportional to file size.".

Comment (enkore) [inline docs/usage.rst]: Yeah

**Comment by ThomasWaldmann** [comment]
merging this now as is, if there is something left to do, we can change it later.

**Comment by ThomasWaldmann** [comment]
fsck, should have cleaned up the fixup first.

**Pull request merged**